src/App.tsx

```diff
@@ -1,14 +1,76 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import Filter from './types/Filter';
 
 export const App: React.FC = () => {
+  const [data, setData] = useState<Todo[] | []>([]);
+
+  const [activeTodo, setActiveTodo] = useState<Todo | undefined>(undefined);
+  const [query, setQuery] = useState('');
+  const [type, setType] = useState(Filter.ALL);
+
+  const hasTodo = data.length > 0;
```

Comment:
You can rewrite like this
```suggestion
  const hasTodo = !!data.length;
```